### PR TITLE
COMMERCE-1726 replace usage of remote services with dedicated helper classes when in frontstore taglibs

### PR DESCRIPTION
**Pull request by luca-pellizzon**

### CODE REVIEW

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by marco-leo** [comment]
ci:test

**Comment by marco-leo** [comment]
ci:test

File: commerce-taglib/src/main/java/com/liferay/commerce/taglib/servlet/taglib/PriceTag.java [hunk 1]
```diff
@@ -353,30 +361,29 @@ protected void setProductInfo() throws Exception {
...
 		}
 
-		if (_cpInstanceId > 0) {
-			_cpInstance = CPInstanceServiceUtil.getCPInstance(_cpInstanceId);
-		}
-		else {
-			if (cpDefinition.isIgnoreSKUCombinations()) {
-				CPInstanceHelper cpInstanceHelper =
-					ServletContextUtil.getCPInstanceHelper();
+		if (!(_cpInstanceId > 0) && cpCatalogEntry.isIgnoreSKUCombinations()) {
```
Comment (ethanbustad) [inline commerce-taglib/src/main/java/com/liferay/commerce/taglib/servlet/taglib/PriceTag.java:364]: `!(_cpInstanceId > 0)` should be simplified to `(_cpInstanceId <= 0)` or `(_cpInstanceId == 0)` (whichever you like better)

Comment (luca-pellizzon) [reply commerce-taglib/src/main/java/com/liferay/commerce/taglib/servlet/taglib/PriceTag.java:364]: See https://github.com/liferay/com-liferay-commerce/pull/1414

**Pull request merged**